com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -89,16 +79,29 @@ private static IJavaProject getJavaProjectFromName(String projectName) throws Co
      *             CoreException
      */
     private static List<IJavaProject> getJavaProjectFromType(String typeFullyQualifiedName) throws CoreException {
```

Comment:
Normally, typeFullQualifiedName should be fullyQualifiedTypeName

Reply:
fixed.